dbt_subprojects/daily_spellbook/models/_sector/addresses/addresses_info.sql

```diff
@@ -31,240 +30,222 @@
+    SELECT address
+    , array_agg(blockchain) AS blockchains
+    , SUM(executed_tx_count) AS executed_tx_count
+    , MAX(max_nonce) AS max_nonce
+    , MAX_BY(blockchain,max_nonce) AS max_nonce_blockchain
+    , MAX(is_smart_contract) AS is_smart_contract
+    , array_agg(blockchain) FILTER (WHERE is_smart_contract) AS smart_contract_blockchains
+    , MAX(namespace) AS namespace
+    , MAX(name) AS name
+    , MIN_BY(first_funded_by, first_funded_by_block_time) AS first_funded_by
+    , MIN_BY(blockchain, first_funded_by_block_time) AS first_funded_blockchain
+    , MIN(first_funded_by_block_time) AS first_funded_by_block_time
+    , SUM(tokens_received_count) AS tokens_received_count
+    , SUM(tokens_received_tx_count) AS tokens_received_tx_count
+    , SUM(tokens_sent_count) AS tokens_sent_count
+    , SUM(tokens_sent_tx_count) AS tokens_sent_tx_count
+    , MIN(first_transfer_block_time) AS first_transfer_block_time
+    , MAX(last_transfer_block_time) AS last_transfer_block_time
+    , MIN(first_received_block_number) AS first_received_block_number
+    , MAX(last_received_block_number) AS last_received_block_number
+    , MIN(first_sent_block_number) AS first_sent_block_number
+    , MAX(last_sent_block_number) AS last_sent_block_number
+    , SUM(sent_volume_usd) AS sent_volume_usd
+    , SUM(received_volume_usd) AS received_volume_usd
+    , MIN(first_tx_block_time) AS first_tx_block_time
+    , MAX(last_tx_block_time) AS last_tx_block_time
+    , map_union(map_from_entries(array[
+            (blockchain, chain_stats)
+            ])) AS chain_stats
+    , MAX(last_seen) AS last_seen
+    , MAX(last_seen_block) AS last_seen_block
+    FROM (
+        {% for addresses_model in addresses_models %}
+        SELECT '{{ addresses_model[0] }}' AS blockchain
+        , address
+        , executed_tx_count
+        , max_nonce
+        , is_smart_contract
+        , namespace
+        , name
+        , first_funded_by
+        , first_funded_by_block_time
+        , tokens_received_count
+        , tokens_received_tx_count
+        , tokens_sent_count
+        , tokens_sent_tx_count
+        , first_transfer_block_time
+        , last_transfer_block_time
+        , first_received_block_number
+        , last_received_block_number
+        , first_sent_block_number
+        , last_sent_block_number
+        , received_volume_usd
+        , sent_volume_usd
+        , first_tx_block_time
+        , last_tx_block_time
+        , first_tx_block_number
+        , last_tx_block_number
+        , map_from_entries(array[
+            ('last_seen', CAST(last_seen AS varchar))
+            , ('last_seen_block', CAST(last_seen_block AS varchar))
+            , ('executed_tx_count', CAST(executed_tx_count AS varchar))
+            , ('is_smart_contract', CAST(is_smart_contract AS varchar))
+            , ('tokens_sent_count', CAST(tokens_sent_count AS varchar))
+            , ('tokens_received_count', CAST(tokens_received_count AS varchar))
```

Comment:
are these still needed? I believe only `last_seen_block` is used?

Reply:
they aren't needed for the incremental runs but are there to provide some small high level information per chain in addresses.info

Reply:
The request is mainly to keep the schema as clean and focused as possible. 
I'm worried that other chain-specific values become important in the future and then we start adding more columns in there and it becomes messy. 
Once we have it in there it becomes difficult to remove as others start using it.

But I understand your concerns about the view performance, if we can keep it to these 3 columns, it's fine for me.

Reply:
Alright then this PR is ready afaik @0xRobin